test/IRGen/keypaths.sil

```diff
@@ -311,7 +311,7 @@ sil_vtable C2 {}
 // CHECK-SAME: <i32 0x3fffffe>
 
 // CHECK-LABEL: @"generic environment SHRzSHR_r0_l" = linkonce_odr hidden constant
-// CHECK-SAME: i32 8193, i16 2, i8 -128, i8 -128, i32 128
+// CHECK-SAME: i32 24577, i16 2, i8 -128, i8 -128, i32 128
```

Comment:
I believe this is due to a change in the number of generic requirements, caused by mistakenly counting Copyable and Escapable requirements when populating this field: 

https://github.com/swiftlang/swift/blob/41ff7383d95f4968db5959783189b05258212851/lib/IRGen/GenProto.cpp#L4627

We filter out those requirements later on when emitting the rest of the generic signature in further down from there, in `irgen::addGenericRequirements`.